drivers/hmis_external_apis/extensions/hmis/hud/enrollment_extension.rb

```diff
@@ -12,6 +12,42 @@ module EnrollmentExtension
 
         included do
           has_many :external_referrals, class_name: 'HmisExternalApis::AcHmis::Referral', dependent: :destroy
+          has_many :source_postings, **hmis_relation(:HouseholdID), class_name: 'HmisExternalApis::AcHmis::ReferralPosting', inverse_of: :enrollments
+
+          def accept_referral!(current_user:)
+            return unless head_of_household?
+            return unless HmisExternalApis::AcHmis::LinkApi.enabled?
```

Comment:
maybe we should flip these checks?  Not sure it matters, but feels like this only applies ever if `HmisExternalApis::AcHmis::LinkApi.enabled?`

Reply:
Yeah, that was my internal counter argument.  Let's leave it.  Thanks.